.github/workflows/release.yml

```diff
@@ -0,0 +1,90 @@
+name: goreleaser
+
+on:
+  create:
+    tags:
+
+jobs:
+  qovery:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: Fetch tags
+        run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+      - name: Ensure tag match the current version
+        run: |
+          version=$(git tag | sort --version-sort | tail -1)
+          version_without_v=$(echo $version | sed -r 's/v(.+)/\1/')
+          file="cmd/version.go"
+          if [ "v$(grep '// ci-version-check' $file | sed -r 's/.+return\s"(.+)".+/\1/')" != "$version" ] ; then
+              echo "Tag version do not match application version in $file"
+              exit 1
+          fi
+
+          file="charts/digital-mobius/Chart.yaml"
+          if [ "v$(grep 'appVersion' $file | awk '{ print $2 }')" != "$version" ] ; then
+           echo "Tag appVersion do not match application version in $file"
+           exit 1
+          fi
+          if [ "v$(grep 'version' $file | awk '{ print $2 }')" != "$version" ] ; then
+           echo "Tag version do not match application version in $file"
+           exit 1
+          fi
+
+          file="charts/digital-mobius/values.yaml"
+          if [ "$(grep 'digitalMobiusImageTag' $file | awk -F'\"' '{ print $2 }')" != "$version_without_v" ] ; then
+            echo "Tag version do not match application version in $file"
+            exit 1
+          fi
+      -
+        name: Set up Go
+        uses: actions/setup-go@main
+        with:
+          go-version: 1.15.x
+      -
+        name: Run golangci-lint
+        uses: actions-contrib/golangci-lint@v1
+        env:
+          GOROOT: ""
+        with:
+          args: "run --timeout 5m0s"
+      -
+        name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v1
+        with:
+          version: latest
+          args: release --rm-dist
+        env:
+          GITHUB_TOKEN: ${{ secrets.GORELEASER_GITHUB_TOKEN }}
+      -
```

Comment:
You should stop here if you don't want to have problems for not yet handled stuffs

Reply:
can I handle them now ?